salesagility/SuiteCRM
# Hotfix 8980

## Description





## Motivation and Context

Cannot download files if extending Documents bean, without this fix.

## How To Test This


1. Extend Documents by putting new file in custom/modules/Documents/Document.php:
```
<?php
if (!defined('sugarEntry') || !sugarEntry) {
    die('Not A Valid Entry Point');
}
require_once 'modules/Documents/Document.php';


class CustomDocument extends Document
{
}
```
2. Add mapping file: to custom/Extension/application/Ext/Include/customDocumentMapping.php
`<?php List['Documents'] = 'Document'; $beanList['Documents'] = 'CustomDocument'; $beanFiles['CustomDocument'] = 'custom/modules/Documents/Document.php';`
3. Run repair from admin console
4. Find a document you would like to download All->Documents (Formal)
5. Click on one of the file names in the list view
6. Check that it downloaded
## Types of changes


### Final checklist
